src/list_of_files.cmake

```diff
@@ -0,0 +1,330 @@
+set(fortran_src 
+  ${CMAKE_CURRENT_SOURCE_DIR}/adn30.f
```

Comment:
`${CMAKE_CURRENT_SOURCE_DIR}/` is not required. These are in the same directory as the current directory.

Reply:
Agreed.